### PR TITLE
Add Row Editing Styling samples

### PR DESCRIPTION
**Pull request by ViktorSlavov**
Related to https://github.com/IgniteUI/igniteui-angular/pull/5436

### CODE REVIEW

File: src/app/grid/grid-row-editing-style/grid-row-editing-style.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            </ng-template>
+        </igx-column>
+        <igx-column field="OrderDate" [dataType]="'date'"></igx-column>
+        <igx-column field="Discontinued" header="Discontinued" [dataType]="'boolean'"></igx-column>
+        <ng-template igxRowEditText let-rowChangesCount>
+            <span class="changes"> Changes: {{rowChangesCount}}</span>
+
+        </ng-template>
+        <ng-template igxRowEditActions let-endRowEdit>
+            <div class="button-wrapper">
```
Comment (IvayloG) [inline src/app/grid/grid-row-editing-style/grid-row-editing-style.component.html]: Format - remove empty space.

File: src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <igx-tree-grid #treeGrid [data]="data" primaryKey="ID" foreignKey="ParentID" width="100%" height="500px"
+        rowEditable=true rowSelectable=true columnHiding=true>
+        <igx-column *ngFor="let c of columns" [editable]="c.editable" [field]="c.field" [dataType]="c.dataType"
+            [header]="c.label" [movable]="c.movable" [resizable]="c.resizable" [sortable]="c.sortable"
+            [filterable]="c.filterable">
+        </igx-column>
+        <ng-template igxRowEditText let-rowChangesCount>
+            <span class="changes"> Changes: {{rowChangesCount}}</span>
+        </ng-template>
+        <ng-template igxRowEditActions let-endRowEdit>
```
Comment (IvayloG) [inline src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html]: The icon buttons from the template do not apply for the igxTree sample and the default ones are actually rendered. Can you please take a look what could be causing it?

File: src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <div class="button-wrapper">
```
[the remark this answers is not in the document]
Comment (ViktorSlavov) [reply src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html]: Yup, removed - thanks for pointing it out!

File: src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html (hunk 2 above)
Comment (ViktorSlavov) [reply src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html]: Thanks for catching this! I've logged an issue in the [main repo](https://github.com/IgniteUI/igniteui-angular/issues/5431)

**Comment by IvayloG** [comment]
Tested with the tree-grid row edit template fix: https://github.com/IgniteUI/igniteui-angular/pull/5436

**Pull request merged**